backend/schemas/common.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+from typing import Generic, Literal, Optional, TypeVar
+
+from pydantic import BaseModel
+
+T = TypeVar('T')
+
+
+class ResponseModel(BaseModel, Generic[T]):
+    status: Literal['success', 'error']
+    message: Optional[str] = None
+    data: Optional[T] = None
+    timestamp: datetime = datetime.now()
```

Comment:
我覺得有做一個common的response 很不錯
但發現一個問題
這個model的timestamp不會更新
可以試著
`timestamp: datetime = Field(default_factory=datetime.now)`

然後我發現code rabbit其實有發現這問題哈哈


> 1. Using `datetime.now()` as a default value can cause issues:
>    
>    * The timestamp is set at model definition time, not instance creation
>    * This can make testing and serialization more difficult